pkg/base1/test-metrics.js

```diff
@@ -127,4 +137,60 @@ QUnit.test("non-instanced decompression", function (assert) {
     assert.deepEqual(sink.samples, [[10], [10]], "got correct samples");
 });
 
+QUnit.test("interval validation", function (assert) {
```

Comment:
I'd rather we'd used ES6 `assert => {... }` syntax, but not a biggie.

Reply:
Can do, as I'm pushing anyway.